app/src/test/java/com/lokico/PSWind/NOAAUnitTests.java

```diff
@@ -0,0 +1,286 @@
+package com.lokico.PSWind;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Jeff on 3/31/2016.
```

Comment:
In general, comments before a function should reflect the content of the function itself.  This is looking like the start of a "changelog".  History of contributions are found in source control, not the source itself.  Changelogs are a collection of issues resolved from your feature/bug tracker.